examples/localdev/docker-compose.yaml

```
@@ -1,14 +1,17 @@
 # A completely self-contained configuration using a bundled
-# postgres database and a bundled identity provider with one
-# hardcoded user.
+# postgres database and a single hard-coded user.
 #
 # Useful for prototyping or local development with no available
-# infrastructure. Container ports are mapped to the range 3000-3004.
-
+# infrastructure. Container ports are mapped to the range 3000-3003.
+#
+# **NOTE**: If you want to have multiple users or integrate your own
+#           external IdP, please start from the `keycloak` example
+#           instead.
 
 services:
   platform:
-    image: ghcr.io/tenzir/platform:${TENZIR_PLATFORM_VERSION}
+    # image: ghcr.io/tenzir/platform:${TENZIR_PLATFORM_VERSION}
```

This can stay in, that way both building and fetching works.

```suggestion
    image: ghcr.io/tenzir/platform:${TENZIR_PLATFORM_VERSION}
```

---

Makes sense, it's a bit annoying that this means the example will be broken by default until we do the next release, but hopefully it'll be just for a few days.